cla-backend/cla/utils.py

```
@@ -2056,7 +2050,7 @@ def extract_pull_request_number(pull_request_message):
             cla.log.debug(f"{fn} - extracted PR number {pull_request_number} from merge_queue data: {pull_request_message} by matching first '#N'")
             return pull_request_number
         else:
-            cla.log.warning(f"{fn} - error - unable to extract pull request number from message: {pull_request_message}")
+            cla.log.warning(f"{fn} - error - unable to extract pull request number from message")
     except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)